src/send/SendOrInviteButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { StyleSheet } from 'react-native'
+import Button, { BtnSizes } from 'src/components/Button'
+import variables from 'src/styles/variables'
+
+interface Props {
+  hidden: boolean
+  disabled: boolean
+  text: string
+  onPress: () => void
+}
+
+export default function SendOrInviteButton({ hidden, disabled, text, onPress }: Props) {
```

Comment:
how about a separate button for send and invite? Having functions serve a single purpose will make our code easier to reason about

Reply:
Also, does this need to be its own component..? Seems like it's just an alias for Button with a certain style

Reply:
Yeah, I originally thought this was going to be "smart" and derive props internally from data (and also that we were going to use this in multiple places) but it ended up being so simple that it doesn't justify its own component.